internal/rbd/rbd_attach.go

```diff
@@ -71,6 +71,24 @@ const (
 var (
 	hasNBD              = true
 	hasNBDCookieSupport = false
+
+	kernelCookieSupport = []util.KernelVersion{
+		{
+			Version:      5,
+			PatchLevel:   14,
+			SubLevel:     0,
+			ExtraVersion: 0, Distribution: "",
```

Comment:
move Distribution to new line

Reply:
or leave it out completely, maybe only list `Version`, `PatchLevel` and `SubLevel` as three digits for kernel versions are most common